alpine/5/Dockerfile

```diff
@@ -38,14 +38,20 @@ RUN mv /usr/local/etc/php/php.ini-production /usr/local/etc/php/php.ini
 # https://hub.docker.com/r/mlocati/php-extension-installer/tags
 COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
 
+
+# `ttf-dejavu` is Chinese supported fonts
 # Install chromium
+# `ttf-dejavu` is Chinese supported fonts
 RUN set -eux; \
     apk add --no-cache \
     font-isas-misc \
     supervisor \
     mysql-client \
     chromium \
+    tzdata \
     ttf-freefont \
+    font-noto-cjk \
+    glibc-i18n \
```

Comment:
can you explain the need for txdata + glibc ?

Reply:
tzdata is for timing accurate, glibc is for  i18n language support, you can remove tzdata but I believe better keep it.

see explanation from chatgpt below:

````
tzdata is a package that provides the time zone database required by various software for time zone and daylight saving time calculations. It contains data representing the history of local time for many locations around the globe.

Key Functions of tzdata
Time Zone Database:

tzdata contains the comprehensive database of time zone information. This includes historical, current, and future time zone and daylight saving time (DST) information for many locations worldwide.
Standardized Time Calculations:

It enables consistent and accurate time zone conversions and calculations across different systems and applications. This is essential for tasks like scheduling events, logging timestamps, and other time-related operations.
Support for Various Applications:

Many software systems and programming languages rely on tzdata for correct time zone handling. For instance, databases, web servers, and various application frameworks use this data to ensure their time-related functionality is accurate.
````



```
glibc-i18n is a component of the GNU C Library (glibc) that focuses on internationalization (i18n) and localization (l10n) support. The GNU C Library is a widely-used standard C library that provides many fundamental system functions and services, and glibc-i18n specifically handles tasks related to making applications suitable for various languages and cultural conventions.

Key Features
Character Set Support:

glibc-i18n supports various character encodings, such as UTF-8, ISO-8859-1, and others, ensuring that applications can handle different character sets.
Localization:

It allows customization of application behavior based on locale settings, such as date formats, currency symbols, and number formats.
Message Translation:

Supports the GNU gettext framework, which enables applications to display messages in different languages based on user preferences.```